source/_integrations/zwave_js.markdown

```diff
@@ -870,12 +870,10 @@ Many users have reported issues with interference when the USB stick was directl
 
 #### How to access the Z-Wave logs
 
-Z-Wave JS writes details to its logs. To access these logs, follow these steps:
+To enable Z-Wave JS logging in the Home Assistant logs, set the log level for `zwave_js_server` to `debug`. This can either be done in your `configuration.yaml` in the `logger` section, or using the `logger.set_level` service. When the integration detects that the log level for `zwave_js_server` has been set to `debug`, it will also set the Z-Wave JS logs to `debug` if its log level isn't already `verbose`, `debug`, or `silly` and will include those logs in the Home Assistant logs. The Z-Wave JS logs can be found under the logger name `zwave_js_server.server`.
```

Comment:
We now have enable debug logging on the integration pages, all this fiddling with configs or services isn't needed anymore.

Reply:
While true, a user may want to enable this logging level on startup since not all issues occur while you're watching.

Reply:
We can describe that additionally. Sending users into something advanced as YAML, will not help in general.